src/pages/TabDetails.tsx

```diff
@@ -186,40 +222,102 @@ const TabDetails = () => {
   })}`;
 
   const p_style = {
-    fontSize: "10px" as const,
-    color: "var(--ion-color-basic)" as const,
-    textAlign: "center" as const,
+    fontSize: "13px" as const,
+    color: "var(--ion-color-light)" as const,
+    textAlign: "left" as const,
+    marginBottom: "5px" as const,
   };
 
   const h_style = {
-    fontWeight: "bold" as const,
-    color: "var(--ion-color-dark-basic)" as const,
-    textAlign: "center" as const,
+    fontSize: "20px" as const,
+    color: "var(--ion-color-light)" as const,
+    textAlign: "left" as const,
+  };
+
+  const History = () => {
+    if (cycles.length === 0) {
+      return (
+        <IonList>
+          <div id="progress-block">
+            <p style={{ fontSize: "13px" }}>
+              {t("You haven't marked any periods yet")}
+            </p>
+          </div>
+          <div style={{ marginTop: "40px" }}>
+            <IonButton
+              class="mark-button"
+              color="dark-basic"
+              onClick={() => props.setIsMarkModal(true)}
+            >
+              {t("mark")}
+            </IonButton>
+            <MarkModal
+              isOpen={props.isMarkModal}
+              setIsOpen={props.setIsMarkModal}
+            />
+          </div>
+        </IonList>
+      );
+    } else if (cycles.length === 1) {
+      return (
+        <IonList>
+          <CurrentCycle />
+        </IonList>
+      );
+    } else {
+      return (
+        <IonList>
+          <CurrentCycle />
+          <ListProgress />
```

Comment:
You can rewrite this else block as

```jsx
<IonList>
  <CurrentCycle />
  {cycles.length > 1 && <IonList />}
</IonList>
```

and remove block above

```jsx
else if (cycles.length == 1)
```

Reply:
Fixed